x-pack/test_serverless/functional/test_suites/examples/config.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { REPO_ROOT } from '@kbn/repo-info';
+import { findTestPluginPaths } from '@kbn/test';
+import { resolve } from 'path';
+import { createTestConfig } from '../../config.base';
+
+export default createTestConfig({
+  serverlessProject: 'es',
```

Comment:
Nope, not necessary and I can move them. I did it this way mainly to have the run once vs three times as they would in `common`, but technically there's no reason why these tests can't all run in each project type. I like the idea of moving them to `common`, but I'm not quite sure how to make that work config wise.

I'm thinking I could move `examples/index.ts` to `common/index.examples.ts` and then update each project type's `config.ts` with these changes so the example tests run:
```ts
export default createTestConfig({
  ...,
  testFiles: [
    require.resolve('../common'),
    require.resolve('../common/index.examples.ts'),
    require.resolve('.'),
  ],
  serverArgs: findTestPluginPaths([
    resolve(REPO_ROOT, 'examples'),
    resolve(REPO_ROOT, 'x-pack/examples'),
  ]),
});
```

I think this would allow the example tests to run locally for each project type while skipping MKI by keeping the `skipMKI` tag in `index.examples.ts`. WDYT?

Reply:
I think we'd like to avoid the example plugin loading where it's not needed in order to avoid potential side effects in other tests which don't explicitly require it (even though the probability of such side effects might be low). That's why I'd suggest to create separate `config.examples.ts` entries in all three projects and make them load the `common/index.examples.ts`

Reply:
Ah ok, I see what you mean now. Sure I can go with that approach then, I should be able to make these changes sometime today.

Reply:
@pheyos Updated the PR with your suggestions.